theme/footer.html

```diff
@@ -32,7 +32,8 @@
                 <h4>Contacts</h4>
                 <div class="hline-w"></div>
                 <p>
-                    <a href="irc://irc.freenode.net/zftalk" class="btn-social btn-outline" alt="IRC" title="IRC"><i class="fa fa-comment"></i></a>
+                    <a href="https://discourse.zendframework.com" class="btn-social btn-outline" alt="Forum" title="Forum"><i class="fa fa-comment"></i></a>
```

Comment:
My suggestion for forum: http://fontawesome.io/icon/comments/

Reply:
Changed:

<img width="252" alt="screen shot 2017-09-21 at 09 35 14" src="https://user-images.githubusercontent.com/7423207/30686465-41f5782c-9eb0-11e7-9434-f2eb65b9e9d7.png">

Reply:
Looks good! (Background: more than one person give comments! :wink: )